test/e2e/app-dir/app-static/app-static.test.ts

```diff
@@ -353,6 +354,19 @@ createNextDescribe(
           )
         }
       })
+
+      it('should properly error when dynamic = "error" page uses dynamic', async () => {
+        const res = await next.fetch('/dynamic-error/static-bailout-1')
+        const outputIndex = next.cliOutput.length
+
+        expect(res.status).toBe(500)
+
+        if (isNextStart) {
```

Comment:
If the page is `dynamic = 'error'` we should be showing the error message in development

Reply:
CI didn't fail so we don't do that now. Can we merge this and create the dev view in a new PR?

Reply:
Can be a follow-up yeah